lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -84,6 +84,29 @@ class CategoryRenderer {
     this.dom.find('.lh-audit__description', auditEl)
         .appendChild(this.dom.convertMarkdownLinkSnippets(audit.result.description));
 
+    if (audit.result.stackPacks) {
+      Object.entries(audit.result.stackPacks).forEach(([name, pack]) => {
```

Comment:
I just added a bit of inline style to get the report look a bit better so please don't judge 😆 🙏

Reply:
its a good time to redo the styling properly, IMO. :)